### DISCUSSION TITLE
Can I turn off hotkeys on the player?

### DISCUSSION THREAD

**Discussion by theDestI** [remotion-dev/remotion#1165] [General]
The title basically says it all

**Comment by JonnyBurger** [comment]
You can set `spaceKeyToPlayAndPause={false}`, which I believe is the only keyboard shortcut we have currently.

Sorry for the late response! Discussions often go under for me for some reason 🤔

**Comment by theDestI** [comment]
To be fair, it didn't help. Besides, there are other hotkeys at the preview player like "A" (go to start), "E" (go to the end), "O" (highlight specific section)

**Reply by JonnyBurger** [reply]
I see - we need to distinguish between the embeddable player (we call it "Remotion Player") and the preview editor (we call it "Remotion Preview").

For the preview, you can indeed disable the hotkeys. I'm wondering about your usecase - why do you want to turn it off, are you accidentially pressing those keys?

**Reply by theDestI** [reply]
I am talking about the preview editor in that case. 

My use case is that I've added two codemirror editors on top of the existing Remotion editor, and this tool allows me to turn CSS+HTML snippets into TikTok videos for my own channel. Sometimes I need to make the "last minute" changes to the code in the editors, and here hotkeys are not helping much.

**Reply by JonnyBurger** [reply]
I see - how are you adding the preview to your webapp? Usually the preview gets configured using the `remotion.config.ts` file, would an option there help you?

**Reply by theDestI** [reply]
Yes, I'm using remotion.config.ts for the preview right now

**Comment by JonnyBurger** [comment]
Implemented in the next version: https://github.com/remotion-dev/remotion/pull/1269

**Comment by theDestI** [comment]
It's great! Thanks a lot
